cmd/syncthing/monitor.go

```diff
@@ -181,6 +186,43 @@ func copyStderr(stderr io.ReadCloser, dst io.Writer) {
 					panicFd.WriteString(line)
 				}
 				stdoutMut.Unlock()
+
+				if optPanicReporting {
+					// Prepare query for collecting error log
+					type Report struct {
+						Error       string
+						Version     string
+						LongVersion string
+						Os          string
+						Arch        string
+					}
+					report := &Report{Error: line[:len(line)-1],
```

Comment:
Then that's probably part of the interesting stuff, as it leads up to why we crashed... But I see what you're getting at.